pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/bestpractices/UnusedPrivateFieldRule.java

```diff
@@ -27,6 +31,17 @@
 
 public class UnusedPrivateFieldRule extends AbstractLombokAwareRule {
 
+    /**
+     * Fields, that are annotated with the annotation in the set
+     * should not be considered by the rule UnusedPrivateField.
+     */
+    private static final Set<String> NEGLECT_ANNOTATIONS = new HashSet<>();
```

Comment:
This should probably be made into a property (`TypeMultiProperty.named("ignoredAnnotations")...`) for ease of configuration.